backend/tests/core/test_exceptions.py

```
@@ -0,0 +1,143 @@
+@pytest.mark.unit
+class TestExceptionHandlers:
+    """Test exception handler functions"""
+
+    async def test_http_exception_handler(self):
+        """Test HTTP exception handler returns correct JSON response"""
+        exc = NotFoundException(detail="User not found")
+
+        response = await http_exception_handler(request=None, exc=exc)
+
+        assert response.status_code == status.HTTP_404_NOT_FOUND
+        response_body = eval(response.body.decode())
```

Ruff (0.14.5)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)